quickstart/101-azfw-with-fwpolicy/variables.tf

```diff
@@ -13,7 +13,7 @@ variable "resource_group_name_prefix" {
 variable "firewall_sku_tier" {
   type        = string
   description = "Firewall SKU."
-  default     = "Premium" # Valid values are Standard and Premium
+  default     = "Standard" # Valid values are Standard and Premium
```

Comment:
Why modify this `fwpolicy` example code in this pr?

Reply:
@lonegunmanb because the example code i got it from was for standard firewall and this example is for premium

Reply:
The changes you've made in this pr are mainly focus in `201-azfw-multi-addresses` folder, and you pr's title showed that this is a new example to demonstrate how to create firewall with multiple addresses. I think maybe it's better to open a separate PR to modify `101-azfw-with-fwpolicy` code, since it's irreverent to this multiple addresses example to me. @cshea15

Reply:
ohh i missed that this was the wrong file, I must have had a copy-and-paste error. thanks for catching this. I will fix the 101-azfw-with-fwpolicy.

Reply:
@lonegunmanb i fixed the error and this should no longer be in this PR. thanks again for catching this.